src/utils/SelectorsUtils.js

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export function getIentifierFromUserEmail(agentName) {
```

Comment:
Could we rename `agentName` to `userEmail` to be consistent with the function name ?